tests/suites/test_suite_rsa.function

```diff
@@ -1129,11 +1129,11 @@ void mbedtls_rsa_export_raw( data_t *input_N, data_t *input_P,
                              int successive )
 {
     /* Exported buffers */
-    unsigned char bufNe[1000];
-    unsigned char bufPe[1000];
-    unsigned char bufQe[1000];
-    unsigned char bufDe[1000];
-    unsigned char bufEe[1000];
+    unsigned char bufNe[256];
+    unsigned char bufPe[128];
+    unsigned char bufQe[128];
+    unsigned char bufDe[256];
+    unsigned char bufEe[1];
```

Comment:
Why these sizes? How were they determined?

Reply:
I ran the tests on PC and printed the values.

Reply:
These are the lengths from the *current* tests:
~~~
RSA PKCS1 Sign #1 (SHA512, 1536 bits RSA) ......................... test_mbedtls_rsa_pkcs1_sign output length is 192
PASS
RSA PKCS1 Sign #1 Verify .......................................... PASS
RSA PKCS1 Sign #2 (SHA256, 2048 bits RSA) ......................... test_mbedtls_rsa_pkcs1_sign output length is 256
PASS
RSA PKCS1 Sign #2 Verify .......................................... PASS
RSA PKCS1 Sign #2 Verify (Fail) ................................... PASS
RSA PKCS1 Sign #3 (SHA224, 2048 bits RSA) ......................... test_mbedtls_rsa_pkcs1_sign output length is 256
PASS
RSA PKCS1 Sign #3 Verify .......................................... PASS
RSA PKCS1 Sign #4 (SHA384, 2048 bits RSA) ......................... test_mbedtls_rsa_pkcs1_sign output length is 256
PASS
RSA PKCS1 Sign #4 Verify .......................................... PASS
RSA PKCS1 Sign #5 (MD2, 2048 bits RSA) ............................ ----
   Unmet dependencies: 4  
RSA PKCS1 Sign #5 Verify .......................................... ----
   Unmet dependencies: 4  
RSA PKCS1 Sign #6 (MD4, 2048 bits RSA) ............................ ----
   Unmet dependencies: 5  
RSA PKCS1 Sign #6 Verify .......................................... ----
   Unmet dependencies: 5  
RSA PKCS1 Sign #7 (MD5, 2048 bits RSA) ............................ test_mbedtls_rsa_pkcs1_sign output length is 256
PASS
RSA PKCS1 Sign #7 Verify .......................................... PASS
RSA PKCS1 Sign #8 (RAW, 2048 bits RSA) ............................ test_rsa_pkcs1_sign_raw output length is 256
test_rsa_pkcs1_sign_raw output length is 256
PASS
RSA PKCS1 Sign #8 Verify .......................................... test_rsa_pkcs1_verify_raw output length is 9
PASS
RSA PKCS1 Sign #8 Verify (Wrong raw hash) ......................... test_rsa_pkcs1_verify_raw output length is 9
PASS
RSA PKCS1 Sign #9 (Invalid Digest type) ........................... test_mbedtls_rsa_pkcs1_sign output length is 256
PASS
RSA PKCS1 Sign #9 Verify (Invalid Digest type) .................... PASS
RSA PKCS1 Sign #8 (Invalid padding type) .......................... test_mbedtls_rsa_pkcs1_sign output length is 256
PASS
RSA PKCS1 Sign #8 Verify (Invalid padding type) ................... PASS
RSA PKCS1 Encrypt #1 .............................................. test_mbedtls_rsa_pkcs1_encrypt output length is 256
PASS
RSA PKCS1 Decrypt #1 (Verify) ..................................... test_mbedtls_rsa_pkcs1_decrypt output length is 16
PASS
RSA PKCS1 Encrypt #2 (Data too large) ............................. test_mbedtls_rsa_pkcs1_encrypt output length is 256
PASS
RSA PKCS1 Decrypt #2 (Data too small) ............................. test_mbedtls_rsa_pkcs1_decrypt output length is 0
PASS
RSA PKCS1 Encrypt #3 (Invalid padding mode) ....................... test_mbedtls_rsa_pkcs1_encrypt output length is 256
PASS
RSA PKCS1 Decrypt #3 (Invalid padding mode) ....................... test_mbedtls_rsa_pkcs1_decrypt output length is 0
PASS
RSA PKCS1 Decrypt #4 (Output buffer too small) .................... test_mbedtls_rsa_pkcs1_decrypt output length is 0
PASS
RSA Private (Correct) ............................................. test_mbedtls_rsa_private output1 length is 256
test_mbedtls_rsa_private output1 length is 256
test_mbedtls_rsa_private output1 length is 256
test_mbedtls_rsa_private output2 length is 256
PASS
RSA Private (Data larger than N) .................................. test_mbedtls_rsa_private output1 length is 256
test_mbedtls_rsa_private output1 length is 256
test_mbedtls_rsa_private output1 length is 256
test_mbedtls_rsa_private output2 length is 256
PASS
RSA Public (Correct) .............................................. test_mbedtls_rsa_public output length is 256
test_mbedtls_rsa_public output length is 256
PASS
RSA Public (Data larger than N) ................................... test_mbedtls_rsa_public output length is 256
test_mbedtls_rsa_public output length is 256
PASS
RSA Export Raw (N,P,Q,D,E) ........................................ test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
test_mbedtls_rsa_export_raw bufPe length is 128
test_mbedtls_rsa_export_raw bufQe length is 128
test_mbedtls_rsa_export_raw bufDe length is 256
PASS
RSA Export Raw (N,P,Q,D,E), successive ............................ test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
test_mbedtls_rsa_export_raw bufPe length is 128
test_mbedtls_rsa_export_raw bufQe length is 128
test_mbedtls_rsa_export_raw bufDe length is 256
PASS
RSA Export Raw (N,-,-,D,E) ........................................ test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
test_mbedtls_rsa_export_raw bufPe length is 0
test_mbedtls_rsa_export_raw bufQe length is 0
test_mbedtls_rsa_export_raw bufDe length is 256
PASS
RSA Export Raw (N,-,-,D,E), succesive ............................. test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
test_mbedtls_rsa_export_raw bufPe length is 0
test_mbedtls_rsa_export_raw bufQe length is 0
test_mbedtls_rsa_export_raw bufDe length is 256
PASS
RSA Export Raw (N,P,Q,-,E) ........................................ test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
test_mbedtls_rsa_export_raw bufPe length is 128
test_mbedtls_rsa_export_raw bufQe length is 128
test_mbedtls_rsa_export_raw bufDe length is 0
PASS
RSA Export Raw (N,P,Q,-,E), successive ............................ test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
test_mbedtls_rsa_export_raw bufPe length is 128
test_mbedtls_rsa_export_raw bufQe length is 128
test_mbedtls_rsa_export_raw bufDe length is 0
PASS
RSA Export Raw (N,-,-,-,E) ........................................ test_mbedtls_rsa_export_raw bufNe length is 256
test_mbedtls_rsa_export_raw bufEe length is 1
PASS
RSA PKCS1 Encrypt Bad RNG ......................................... test_rsa_pkcs1_encrypt_bad_rng output length is 256
PASS
~~~